Jenkinsfile

```diff
@@ -3,7 +3,12 @@
 // https://github.com/feedhenry/fh-pipeline-library
 @Library('fh-pipeline-library') _
 
-fhBuildNode {
+node('nodejs4') {
+    stage('Checkout') {
+        step([$class: 'WsCleanup'])
```

Comment:
Is this not normally added above the Checkout stage? Just to be consistent with other Jenkinsfiles. For example: https://github.com/feedhenry/fh-amqp-js/blob/master/Jenkinsfile#L8

Reply:
It is, but it doesn't really matter. I think we should probably tend towards having any actions/steps in stages, and this one kinda fits in this stage. What do you think? Happy to change it.

The reason we've probably done it outside the stage in other ones, is because it was outside the stage in the fhBuildNode method I think

Reply:
I don't feel strongly about it to be honest, was more of an observation than an issue. Happy for you to merge it 👍

Reply:
Thanks!